### PR TITLE
gene coverage in pdf case report

### PR DESCRIPTION
**Pull request by northwestwitch**
After struggling with our firewall i finally managed to include the html code from the coverage report at the bottom of the case report.

I'm reasoning that maybe it's not so important to have both case report and coverage report in the same HTML page, and it's more useful for the users to be able to export them both together in the PDF version of the report, but if it's needed i can include the coverage also in the HTML, it's super easy!

This PR might be already tested from scout stage (scout-stage.scilifelab.se) but since we didn't set up a proper connection with the coverage database there, all the numbers in the table are 0!

### CODE REVIEW

**Comment by northwestwitch** [comment]
fix #978

File: scout/server/blueprints/cases/controllers.py [hunk 1]
```diff
@@ -219,6 +222,55 @@ def case_report_content(store, institute_obj, case_obj):
...
+
+    request_data = {}
+    # extract sample ids from case_obj and add them to the post request object:
+    request_data['sample_id'] = [ ind['individual_id'] for ind in case_obj['individuals'] ]
+
+    # extract default panel names and default genes from case_obj and add them to the post request object
+    distinct_genes = set()
+    panel_name = []
+    for panel_info in case_obj.get('panels', []):
+        if panel_info.get('is_default'):
```
Comment (moonso) [inline scout/server/blueprints/cases/controllers.py]:
```suggestion
        if not panel_info.get('is_default'):
            continue
         panel_obj = store.gene_panel(panel_info['panel_name'], version=panel_info.get('version'))
         full_name = "{} ({})".format(panel_obj['display_name'], panel_obj['version'])
         panel_name.append(full_name)
```

File: scout/server/blueprints/cases/controllers.py [hunk 2]
```diff
@@ -219,6 +222,55 @@ def case_report_content(store, institute_obj, case_obj):
...
+    Returns:
+        coverage_data(str): string rendering of the content between <body </body> tags of a coverage report
+    """
+
+    request_data = {}
+    # extract sample ids from case_obj and add them to the post request object:
+    request_data['sample_id'] = [ ind['individual_id'] for ind in case_obj['individuals'] ]
+
+    # extract default panel names and default genes from case_obj and add them to the post request object
+    distinct_genes = set()
```
Comment (moonso) [inline scout/server/blueprints/cases/controllers.py]: This is not used? If not used, remove

File: scout/server/blueprints/cases/controllers.py [hunk 3]
```diff
@@ -219,6 +222,55 @@ def case_report_content(store, institute_obj, case_obj):
...
+    panel_name = []
```
Comment (moonso) [inline scout/server/blueprints/cases/controllers.py]: panel_names

File: scout/server/blueprints/cases/controllers.py [hunk 4]
```diff
@@ -219,6 +222,55 @@ def case_report_content(store, institute_obj, case_obj):
...
+    for panel_info in case_obj.get('panels', []):
+        if panel_info.get('is_default'):
+            panel_obj = store.gene_panel(panel_info['panel_name'], version=panel_info.get('version'))
+            #distinct_genes.update([gene['hgnc_id'] for gene in panel_obj.get('genes', [])])
+            full_name = "{} ({})".format(panel_obj['display_name'], panel_obj['version'])
+            panel_name.append(full_name)
+    panel_name = ' ,'.join(panel_name)
+    request_data['panel_name'] = panel_name
```
Comment (moonso) [inline scout/server/blueprints/cases/controllers.py]:
```suggestion
    request_data['panel_name'] = panel_names
```

File: scout/server/blueprints/cases/controllers.py [hunk 5]
```diff
@@ -219,6 +222,55 @@ def case_report_content(store, institute_obj, case_obj):
...
+    #post_request_data['default_genes'] = list(distinct_genes)
+
+    # add institute-specific cutoff level to the post request object
+    request_data['level'] = institute_obj.get('coverage_cutoff')
```
Comment (moonso) [inline scout/server/blueprints/cases/controllers.py]:
```suggestion
    request_data['level'] = institute_obj.get('coverage_cutoff', 15)
```

**Comment by moonso** [comment]
This needs to be rebased

File: scout/server/blueprints/cases/controllers.py (hunk 2 above)
Comment (moonso) [reply scout/server/blueprints/cases/controllers.py]: @northwestwitch what are you thinking of this?

Comment (northwestwitch) [reply scout/server/blueprints/cases/controllers.py]: Genes don't need to be passed to chanjo-report, just panels, so it's safe to remove the commented code

**Pull request merged**